sysadmin/systemd/core/ipol-core.service

```diff
@@ -7,6 +7,7 @@ ExecStart=/bin/bash -c "source ./venv/bin/activate && python main.py core.conf"
 WorkingDirectory=/home/ipol/ipolDevel/ipol_demo/modules/core/
 Environment=IPOL_HOST=ipolcore.ipol.im
 Environment=IPOL_URL=https://ipolcore.ipol.im
+Environment=env=production
```

Comment:
Is this variable really used? It would be best if the code didn't care about the deployment target.

Reply:
It is used to discriminate when sending demo failure emails sadly..

Reply:
We will change that with the new db and fastapi backend